lib/phoenix_live_view/tokenizer.ex

```diff
@@ -155,6 +155,18 @@ defmodule Phoenix.LiveView.Tokenizer do
     handle_tag_open(rest, line, column + 1, text_to_acc, %{state | context: []})
   end
 
+  defp handle_text("{" <> rest, line, column, [?\\ | buffer], acc, state) do
```

Comment:
I don't think we should allow escapes. Because if `\{` means `{`, then how do I write `\{`? We would need to support `\\{` and so on. We should probably follow JSX here. If you want to write `{`, you can either write `{ "{" }`, `&#123;` or `&lbrace;`.